shuffle/cli/README.md

```diff
@@ -0,0 +1,37 @@
+## Step 0: Install Dependencies
+
+- Install Diem dependencies including Rust, Clang, etc, by running the following script in `diem` root directory:
+```
+./scripts/dev_setup.sh
+```
+
+## Usage
+
+Please run `cargo run -p shuffle -- help`. Ultimately, this will become
+`shuffle help`.
+
+## Sample Usage
+
+1. `cargo run -p shuffle -- new /tmp/helloblockchain` creates a new shuffle project
+2. `cargo run -p shuffle -- node /tmp/helloblockchain` runs node based on project
+
+## Development
+
+Note that for local development, `shuffle` is replaced with `cargo run -p shuffle --`:
+
+```bash
+shuffle new helloblockchain # is replaced by
+cargo run -p shuffle -- new helloblockchain
```

Comment:
Why are we doing it like this instead of `shuffle new helloblockchain --local` or some thing like that? Or is this just to get it working for now

Reply:
Discussed offline. "local development" refers to a developer working in the diem/diem repo and not devnet.